syntax="proto3";

service NmapService{
	rpc ipScan(IpRequest) returns (IpResponse){};
}

message IpRequest{
	string ip = 10;
}

message IpResponse{
	string up =10;
	string down=20;
	string total=30;
	repeated IpList result=40;
}

message IpList{
	string ip=10;
	string status=20;
	string os=30;
}
